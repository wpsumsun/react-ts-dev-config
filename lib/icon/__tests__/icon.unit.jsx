import * as renderer from 'react-test-renderer'
import React from 'react'
import Icon from '../icon'

describe('Icon', () => {
  it('是个Icon', () => {
    const json = renderer.create(<Icon name='alipay'/>).toJSON()
    expect(json).toMatchSnapshot()
  })
})