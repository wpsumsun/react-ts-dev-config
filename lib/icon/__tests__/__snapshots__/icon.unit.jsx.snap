// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon 是个Icon 1`] = `
<svg
  className="guluer-icon"
>
  <use
    xlinkHref="#alipay"
  />
</svg>
`;
